website/docs/configuration/variables.html.md

```diff
@@ -76,51 +76,35 @@ assign a value to the variable from outside and to reference the variable's
 value from within the module.
 
 The name of a variable can be any valid [identifier](./syntax.html#identifiers)
-_except_ the following:
-
-- `source`
-- `version`
-- `providers`
-- `count`
-- `for_each`
-- `lifecycle`
-- `depends_on`
-- `locals`
+_except_ the following: `source`, `version`, `providers`, `count`, `for_each`, `lifecycle`, `depends_on`, `locals`.
```

Comment:
I'm curious about this change;  these are in-page links on the website (I believe that's what the extra `-` does): 

https://www.terraform.io/docs/configuration/variables.html#declaring-an-input-variable

Reply:
It is solely to make them take up less real-estate on the page. They are links, but you'll note that they just link to themselves ... which adds no extra context. https://www.terraform.io/docs/configuration/variables.html#locals

Reply:
got it, thanks!